components/explore-the-data-page/DataDownloadButton.js

```diff
@@ -41,10 +44,29 @@ class DataDownloadButton extends React.Component {
           download={fileName}
           target="_blank"
           rel="noopener noreferrer"
-          onClick={() => this.startDownload(fileName)}
+          onClick={() => this.setState({ dataLicenseModalOpen: true })}
         >
           Download (CSV)
         </A>
+        {dataLicenseModalOpen && (
+          <Modal
+            button={{ text: 'Accept & Download', clickFunction: () => this.startDownload(fileName) }}
+            onClose={() => this.setState({ dataLicenseModalOpen: false })}
+          >
+            <div>
+              If you use TJI’s data, you must give TJI credit and adhere to TJI’s{' '}
+              <a
+                href="https://github.com/texas-justice-initiative/data-processing"
```

Comment:
```suggestion
                href="https://github.com/texas-justice-initiative/data-processing/blob/master/DataUsageAgreement.md"
```

Reply:
please double-check that that works! https://github.com/texas-justice-initiative/data-processing/blob/master/DataUsageAgreement.md